embeddings/transformation/flair/classification_corpus_transformation.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+import datasets
+from flair.datasets import CSVClassificationDataset
+
+from embeddings.transformation.flair.corpus_transformations import CorpusTransformation
+
+
+class ClassificationCorpusTransformation(CorpusTransformation):
+    def _preprocess_subset(
+        self, hf_datadict: datasets.DatasetDict, subset_name: str, output_path: Path
+    ) -> CSVClassificationDataset:
+        label_map = hf_datadict[subset_name].features[self.target_column_name].names
+        hf_datadict[subset_name] = hf_datadict[subset_name].map(
+            lambda row: {"named_target": label_map[row[self.target_column_name]]},
+            remove_columns=[self.target_column_name],
+        )
+
+        hf_datadict[subset_name].to_csv(
+            output_path.joinpath(f"{subset_name}.csv"), header=False, index=False
+        )
+
+        column_name_map = {
+            hf_datadict[subset_name].column_names.index(self.input_column_name): "text",
```

Comment:
CC: @albert097

Reply:
these names are required by Flair to identify input and label. I see no point to make it constant because it is not my convention of naming and we can not change it.